test/known_issues/test-stdout-buffer-flush-on-exit.js

```diff
@@ -15,7 +15,7 @@ if (process.argv[2] === 'child') {
   process.exit();
 }
 
-[16, 18, 20].forEach((exponent) => {
+[16, 18, 20, 21, 22].forEach((exponent) => {
```

Comment:
should it be adding 22 and 24 to stay consistent here? or are 21 and 22 magic somehow?